manifests/packages.pp

```diff
@@ -2,7 +2,7 @@
 #
 # Sets up packages for openssl
 class openssl::packages {
-  package { 'openssl':
+  package { $openssl::package_name:
```

Comment:
I don't believe that permits passing in a list of packages (such as ['openssl', 'compat-openssl'] or ['openssl', 'openssl.i686']) which is my primary use case.